docs/setup/settings.asciidoc

```diff
@@ -669,3 +669,4 @@ include::{kibana-root}/docs/settings/spaces-settings.asciidoc[]
 include::{kibana-root}/docs/settings/task-manager-settings.asciidoc[]
 include::{kibana-root}/docs/settings/telemetry-settings.asciidoc[]
 include::{kibana-root}/docs/settings/url-drilldown-settings.asciidoc[]
+include::{kibana-root}/docs/settings/ai-assistant-settings.asciidoc[]
```

Comment:
This list appears to be sorted alphabetically. Should this be moved up to the top?

Reply:
I have no idea, but I assume you're right, so moved!